code/modules/vending/autodrobe.dm

```diff
@@ -97,8 +97,11 @@
 				/obj/item/clothing/mask/gas/sexymime = 1,
 				/obj/item/clothing/under/rank/civilian/mime/sexy = 1,
 				/obj/item/clothing/under/rank/civilian/mime/skirt = 1,
-				/obj/item/clothing/under/rank/civilian/clown/jester = 1,
 				/obj/item/clothing/head/costume/jester = 1,
+				/obj/item/clothing/under/rank/civilian/clown/jester = 1,
+				/obj/item/clothing/shoes/clown_shoes/green_jester_shoes =1,
+				/obj/item/clothing/head/costume/jester/alt =1,
+				/obj/item/clothing/under/rank/civilian/clown/jester/alt =1,
```

Comment:
if you could, it would be nice to leave a tag for future coders when you are editing something in the normal code, and probably shouldnt shift the jester outfit up from its normal spot
```suggestion
				/obj/item/clothing/under/rank/civilian/clown/jester/alt =1, // MONKEYSTATION ADDITION
```